src/pkg/notifier/handler/auditlog/handler.go

```diff
@@ -0,0 +1,46 @@
+package auditlog
+
+import (
+	beegoorm "github.com/astaxie/beego/orm"
+	"github.com/goharbor/harbor/src/common/utils/log"
+	"github.com/goharbor/harbor/src/internal/orm"
+	"github.com/goharbor/harbor/src/pkg/audit"
+	am "github.com/goharbor/harbor/src/pkg/audit/model"
+	"github.com/goharbor/harbor/src/pkg/notifier/model"
+)
+
+// Handler
+type Handler struct {
+	AuditLogMgr audit.Manager
+}
+
+type AuditResolver interface {
+	ResolveToAuditLog() (*am.AuditLog, error)
+}
+
+var AuditHandler = Handler{AuditLogMgr: audit.Mgr}
+
+func (h *Handler) Handle(value interface{}) error {
+	ctx := orm.NewContext(nil, beegoorm.NewOrm())
```

Comment:
The `Handle` function accepts the parameter `context` with the `ormer` and `transaction` populated? The `context` should be the same one within the request but populated new `transaction` as some event handling is a asynchronous process